packages/opencensus-core/src/common/console-logger.ts

```diff
@@ -104,19 +101,14 @@ export class ConsoleLogger implements types.Logger {
   }
 }
 
-
-// TODO: reevaluate the need to create a new logger instance on every call to
-// logger();
-
 /**
- *  Function logger exported to others classes.
+ * Function logger exported to others classes. Inspired by:
+ * https://github.com/cainus/logdriver/blob/bba1761737ca72f04d6b445629848538d038484a/index.js#L50
  * @param options A logger options or strig to logger in console
  */
 // tslint:disable-next-line:no-any
 const logger: any = (options?: types.LoggerOptions|string|number) => {
-  const aLogger = new ConsoleLogger(options);
-  logger['logger'] = aLogger;
-  return aLogger;
+  return new ConsoleLogger(options);
```

Comment:
Nit: Make this function have return type `ConsoleLogger`

Reply:
Sure.

Reply:
`types.Logger` is better. Thanks!